### ISSUE TITLE
[REQUEST] Make conversation bubble a little big bigger

### ISSUE THREAD

**Issue by hakiimmislam** [CreatureSurvive/bubbles#12]
From my opinion, if the conversation bubble is slightly bigger like the concept is much more nice.

**Comment by CreatureSurvive** [comment]
I agree, i will be adjusting the cells before it exits the beta phase.

**Comment by hakiimmislam** [comment]
Another one, I noticed that if the message is long, the normal message will show like '...' at the end of the message. It is possible to follow the concept where there will be a break line when the message is long. For example in HTML we use `<br>` or in PHP we use `\n`. Here is the [screenshot](https://imgur.com/a/eHUFkDc).
